api/internal/handler/route/route_test.go

```diff
@@ -985,6 +985,32 @@ func TestRoute(t *testing.T) {
 	_, err = handler.BatchDelete(ctx)
 	assert.Nil(t, err)
 
+	route7 := &entity.Route{}
+	reqBody = `{
+	  "id": "1",
+	  "methods": ["GET"],
+	  "upstream": {
+	      "nodes": {
+	          "httpbin.org:443": 1
+	      },
+	      "type": "roundrobin"
+	  },
+	  "desc": "new route",
+	  "uri": "/get"
+	}`
+	err = json.Unmarshal([]byte(reqBody), route7)
+	assert.Nil(t, err)
+	ctx.SetInput(route7)
+	ret, err = handler.Create(ctx)
+	assert.NotNil(t, err)
+	portTransmit := &ParamsInput{}
+	// FIXME
+	reqBody = `{"url":"http://127.0.0.1:9080/get","method":"GET"}`
```

Comment:
needs @nic-chen's help~

the fixed address is not appropriatly here, however, it is also not appropriate to add a configuration item just for test case, do you have any suggestions?

Reply:
@liuxiran 
I think this should be E2E test, not unit test.

we could finish the job on APISIX then go ahead with this.

Reply:
And I think this PR should be split into two, api and web separate, what do you think?

Reply:
thank for your advice @nic-chen , let me try to update the test case, and split the pr.